test_util/test_upgrade_vpc.py

```diff
@@ -173,35 +268,11 @@ def main():
 
     cluster_api.wait_for_dcos()
 
-    # Deploy an app, and make sure deployments are completely done.
-    cluster_api.marathon.deploy_app(get_test_app())
-
-    cluster_api.marathon.ensure_deployments_complete()
-
-    task_info_before_upgrade = get_task_info(cluster_api.marathon.get('v2/apps').json(),
-                                             cluster_api.marathon.get('v2/tasks').json())
-
-    assert task_info_before_upgrade is not None, "Unable to get task details of the cluster."
-    assert task_info_before_upgrade.state == "TASK_RUNNING", "Task is not in the running state."
-
     with cluster.ssher.tunnel(cluster.bootstrap_host) as bootstrap_host_tunnel:
         bootstrap_host_tunnel.remote_cmd(['sudo', 'rm', '-rf', cluster.ssher.home_dir + '/*'])
 
-    test_util.cluster.upgrade_dcos(cluster, installer_url, add_config_path=config_yaml_override_upgrade)
-
-    task_info_after_upgrade = get_task_info(cluster_api.marathon.get('v2/apps').json(),
-                                            cluster_api.marathon.get('v2/tasks').json())
-
-    assert task_info_after_upgrade is not None, "Unable to get the tasks details of the cluster."
-    assert task_info_after_upgrade.state == "TASK_RUNNING", "Task is not in the running state."
-
-    assert task_info_before_upgrade.id == task_info_after_upgrade.id, \
```

Comment:
I see the test case of tasks ID before and after are same covered

🔴 Does the cluster_workload cover these other tests indirectly?

* Verify Tasks are in RUNNING state after the upgrade. Is it redundant with previous test ?
* Verify there has happened at least one health-check after the upgrade. (This was actually @BenWhitehead  suggestion that we test it.).